### PR TITLE
Cleaning up the Readme file

### PR DESCRIPTION
**Pull request by eltonmesquita**
The readme.md was a bit clutched, with some repeated instructions, messy writing, dated info and useless examples. I've cleaned it up a bit and hopefully improved the copy.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -98,25 +74,22 @@ Full example:
...
-```html
-<head>
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css">
-</head>
-```
 
-You can do a whole bunch of other stuff with animate.css when you combine it with jQuery or add your own CSS rules. Dynamically add animations using jQuery with ease:
+## Usage with jQuery
+
+You can do a whole bunch of other stuff with animate.css when you combine it with jQuery. A simple example:
```
Comment (daneden) [inline README.md]: We should probably replace this with vanilla JS examples soon, but I'm ok with punting on that until a later date.

File: README.md [hunk 2]
```diff
@@ -98,25 +74,22 @@ Full example:
...
+```css
+.yourElement {
+  animation-duration: 3s;
+  animation-delay: 2s;
+  animation-iteration-count: infinite;
+}
 ```
 
-or use the version hosted by [CDNJS](https://cdnjs.com/libraries/animate.css)
+_Note: be sure to replace "vendor" in the CSS with the applicable vendor prefixes (webkit, moz, etc)._
```
[the remark this answers is not in the document]
Comment (eltonmesquita) [reply README.md]: Alright. Removing it 😄

File: README.md (hunk 1 above)
Comment (eltonmesquita) [reply README.md]: You are completely right. I'll do it as soon as I get a bit of free time.

Comment (eltonmesquita) [reply README.md]: As it's better to merge this PR asap, let's move it on #875, alright?

Comment (daneden) [reply README.md]: Perfect!

**Pull request merged**